DependencyInjection/Configuration.php

```diff
@@ -208,4 +210,56 @@ private function addVisitorsSection(NodeBuilder $builder)
             ->end()
         ;
     }
+
+    private function addContextSection(NodeBuilder $builder)
+    {
+        $builder
+            ->arrayNode('context')
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->booleanNode('serialize_null')
+                        ->defaultFalse()
+                        ->treatNullLike(false)
+                        ->info('Flag if null values should be serialized')
+                    ->end()
+                    ->arrayNode('attributes')
+                        ->fixXmlConfig('attribute')
+                        ->defaultValue([])
+                        ->treatNullLike([])
+                        ->useAttributeAsKey('key')
+                        ->prototype('scalar')->end()
+                        ->info('Arbitrary key-value data for context')
+                    ->end()
+                    ->arrayNode('groups')
+                        ->fixXmlConfig('group')
+                        ->defaultValue([GroupsExclusionStrategy::DEFAULT_GROUP])
+                        ->treatNullLike([GroupsExclusionStrategy::DEFAULT_GROUP])
+                        ->prototype('scalar')->end()
+                        ->info('Default serialization groups')
+                    ->end()
+                    ->scalarNode('version')
+                        ->defaultNull()
+                        ->validate()
+                            ->always(function ($value) {
+                                if ($value === null) {
+                                    return null;
+                                }
+
+                                if (!is_string($value)) {
+                                    throw new InvalidArgumentException('Version must be a string.');
+                                }
+
+                                if (!preg_match('#\d+\.\d+\.\d+#', $value)) {
```

Comment:
I will avoid this extra regex... just becase version schemes change often.

In a project in the past I've worked, versions were just one digit.